rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -260,12 +235,12 @@ namespace
 			mip = D3D12_FILTER_TYPE_LINEAR;
 			return;
 		case CELL_GCM_TEXTURE_CONVOLUTION_MIN:
-			LOG_WARNING(RSX, "CELL_GCM_TEXTURE_CONVOLUTION_MIN not supported, fallback to bilinear filtering");
 			min = D3D12_FILTER_TYPE_LINEAR;
 			mip = D3D12_FILTER_TYPE_POINT;
 			return;
 		}
-		unreachable("Wrong min filter");
+		unreachable("Unimplemented min filter used %x, please report this to a developer.", min_filter);
+
```

Comment:
@Zangetsu38 I think Google Translate would be better. Since currently your reading and writing comprehension is on the level of a 4th or 5th grader in my country. I don't still fully understand, what you're trying to say.